easybuild/easyconfigs/h/Harminv/Harminv-1.4.1-intel-2020a.eb

```
@@ -0,0 +1,26 @@
+easyblock = 'ConfigureMake'
+
+name = 'Harminv'
+version = '1.4.1'
+
+homepage = 'https://github.com/stevengj/harminv'
+description = """Harminv is a free program (and accompanying library) to solve the problem of harmonic inversion -
+ given a discrete-time, finite-length signal that consists of a sum of finitely-many sinusoids (possibly exponentially
+ decaying) in a given bandwidth, it determines the frequencies, decay constants, amplitudes, and phases of those
+ sinusoids."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'opt': True, 'unroll': True, 'optarch': True, 'pic': True, 'cstd': 'c99'}
+
+source_urls = ['https://github.com/stevengj/harminv/releases/download/v%(version)s/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['e1b923c508a565f230aac04e3feea23b888b47d8e19b08816a97ee4444233670']
+
+configopts = "--with-pic  --with-blas=mkl_em64t --with-lapack=mkl_em64t --enable-shared"
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['bin', 'lib']
```

The sanity checks can be improved
```suggestion
    'files': ['bin/harminv', 'lib/libharminv.a', 'lib/libharminv.%s' % SHLIB_EXT],
    'dirs': ['include']
```